README.md

```diff
@@ -1,6 +1,11 @@
 three.js
 ========
 
+[![Latest NPM release][npm-badge]][npm-badge-url]
+[![License][license-badge]][license-badge-url]
+[![Dependencies][dependencies-badge]][dependencies-badge-url]
+[![Dev Dependencies][devDependencies-badge]][devDependencies-badge-url]
```

Comment:
Regarding `dependences`, `devDependencies`... or both? We have no `dependences`, so I can totally see not needing that.